app/assets/stylesheets/components/_ajax-loader.scss

```diff
@@ -0,0 +1,60 @@
+$loader-color: $color-black;
+$loader-size: 48px;
+$loader-size-half: $loader-size / 2;
+$number-of-circles: 8;
+$circle-size: $loader-size / $number-of-circles;
+$circle-size-half: $circle-size / 2;
+$fourtyfiveconst: (1 - .70711) * $loader-size-half;
```

Comment:
Not gonna go too in depth with this one as we've been through it in person, but could we have this named a bit more descriptively?

Like `$fourty-five-degrees-in-pixels` or whatever it actually is?